packages/web-components/src/components/va-checkbox/va-checkbox.tsx

```diff
@@ -230,14 +256,15 @@ export class VaCheckbox {
             aria-describedby={ariaDescribedbyIds}
             aria-invalid={error ? 'true' : 'false'}
             disabled={disabled}
+            data-indeterminate={indeterminate && !checked}
```

Comment:
The USWDS release notes say:

> when you set `input.indeterminate = true` via JavaScript **or** add the `data-indeterminate` attribute

...but I couldn't get everything working between styles and screenreader support without adding both `indeterminate` using JS and also adding this `data-indeterminate` attribute. Maybe this is a shadow dom limitation?